samples/instance.js

```diff
@@ -91,6 +95,16 @@ require('yargs')
   .example(
     'node $0 createInstanceWithProcessingUnits "my-instance" "my-project-id"'
   )
+  .command(
```

Comment:
Don't need this. New samples don't follow this structure.

Reply:
Removed